applications/RANSModellingApplication/tests/ChannelFlowTest/channel_flow_k_epsilon_transient_mpi_parameters.json

```diff
@@ -0,0 +1,396 @@
+{
+    "problem_data": {
+        "problem_name": "geometry",
+        "parallel_type": "MPI",
```

Comment:
shouldn't this be the only difference btw OMP and MPI => why a complete new set of settings is needed for the test?

Reply:
in mpi I use different liner solvers, that is the reason to have different json files